### PR TITLE
hw2 ex1 initial

### PR DESCRIPTION
**Pull request by igor-korost**

### CODE REVIEW

File: src/test/java/src/test/java/ru/training/at/hw2/ex1/SoftAssertsTest.java [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    @Test
+    public void test() {
+        // Use WebDriverManager to manage browser driver
+        WebDriverManager.chromedriver().setup();
+        // Create new instance of WebDriver
+        webDriver = new ChromeDriver();
+        // implicit waits
+        webDriver.manage().timeouts().implicitlyWait(2, TimeUnit.SECONDS);
+        // Start browser maximized
+        webDriver.manage().window().maximize();
```
Comment (DmitryKhodakovsky) [inline src/test/java/src/test/java/ru/training/at/hw2/ex1/SoftAssertsTest.java]: Could be extracted to the `@BeforeMethod` of `BaseClass`

File: src/test/java/src/test/java/ru/training/at/hw2/ex1/SoftAssertsTest.java [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+        // 1. Open test site by URL
+        String baseUrl = "https://jdi-testing.github.io/jdi-light/index.html";
```
Comment (DmitryKhodakovsky) [inline src/test/java/src/test/java/ru/training/at/hw2/ex1/SoftAssertsTest.java]: Could be extracted to the constants in the base class

File: src/test/java/src/test/java/ru/training/at/hw2/ex1/SoftAssertsTest.java [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+        softAssert.assertTrue(navElementsStringList.contains("SERVICE"));
+        softAssert.assertTrue(navElementsStringList.contains("METALS & COLORS"));
+        softAssert.assertEquals(navStringCount, 4);
+        softAssert.assertTrue(isDisplayedNav);
+
+        // 6. Assert that there are 4 images on the Index Page and they are displayed
+        WebElement iconPractise = webDriver.findElement(By.cssSelector(".icon-practise"));
+        WebElement iconCustom = webDriver.findElement(By.cssSelector(".icon-custom"));
+        WebElement iconMulti = webDriver.findElement(By.cssSelector(".icon-multi"));
+        WebElement iconBase = webDriver.findElement(By.cssSelector(".icon-base"));
```
Comment (DmitryKhodakovsky) [inline src/test/java/src/test/java/ru/training/at/hw2/ex1/SoftAssertsTest.java:66]: icons could be find as `List<webElement>` and through the loop could be checked for the `isDisplayer`

**Review by DmitryKhodakovsky — CHANGES_REQUESTED** [review]
Where is the ex2? 
Without ex2 work will not accepted

File: src/test/java/src/test/java/ru/training/at/hw2/ex1/SoftAssertsTest.java (hunk 3 above)
Comment (igor-korost) [reply src/test/java/src/test/java/ru/training/at/hw2/ex1/SoftAssertsTest.java:66]: Icons common parent (ancestor) is too far away to get them by findElements(). The code will be too cumbersome. And I don't know how to check whether I got the right icons in my List<WebElement>. For String Lists, I can simply print them with println().

**Review by DmitryKhodakovsky — APPROVED** [review]

**Pull request merged**